.github/workflows/render-rfcs-pr.yaml

```diff
@@ -0,0 +1,51 @@
+name: Render RFCs and create/update PR on changes
+
+on:
+  schedule:
+    - cron: "0 7-18 * * MON-FRI"
+
+  push:
+
+jobs:
+  render-rfcs-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          path: handbook
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: 'giantswarm/rfc'
+          path: rfc
+
+      - name: Render RFCs
+        env:
+          RFCS_DIR: "${{ github.workspace }}/rfc"
+        run: |
+          make rfcs
+
+          # https://github.com/peter-evans/create-pull-request/issues/783
+          sudo chown -R $USER:$USER content/docs/rfcs
+        working-directory: "${{ github.workspace }}/handbook"
+
+      - name: Create or update pull request
+        uses: peter-evans/create-pull-request@v5
+        if: ${{ github.ref == 'refs/heads/main' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
Which token is this? Because the commit looked like it was yours @AndiDog

Reply:
Maybe the plan is to use `${{ secrets.GITHUB_TOKEN }}` and smbd accidentally pasted the line from a repo which uses a dedicated token?

Reply:
I had copy-pasted from the action README, I guess. But it's working, so it might be an old syntax in GitHub actions. Or do you know more? I'll change it in the follow-up PR and then we see if it still works.